app/controllers/organizer/session_types_controller.rb

```diff
@@ -48,7 +46,12 @@ def destroy
   private
 
   def set_session_type
-    @session_type = @event.session_types.find(params[:id])
+    if params[:id]
```

Comment:
Oh just saw this. I was thinking of it as a way to flatten out the controller expectation of @session_type being set for all actions that use it. I figured it was either this or write a separate before_action for the 'new' case. Not a terribly important thing I suppose.